operators/deploy/custom-metrics-server/templates/daemonset.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "custom-metrics-server.fullname" . }}
+  labels:
+    {{- include "custom-metrics-server.labels" . | nindent 4 }}
+{{- if .Values.daemonsetAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+{{- end }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "custom-metrics-server.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- if .Values.podAnnotations }}
+      annotations:
+        {{ toYaml .Values.podAnnotations | indent 8 }}
+    {{- end }}
+      labels:
+        {{- include "custom-metrics-server.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{ toYaml .Values.imagePullSecrets | indent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "exam-agent.fullname" . }}
+      tolerations:
+        {{ toYaml .Values.tolerations | indent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ include "custom-metrics-server.version" . }}"
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
```

Comment:
Should be defined in the values file.

Reply:
imagePullPolicy should be already there. Or are you referring to something else?